### PR TITLE
Relaties operations added

### PR DESCRIPTION
**Pull request by floris85**
Relaties operations added

Also made some things public, to be able to access them from external code.

### CODE REVIEW

File: SnelStart.B2B.Client/Operations/Relaties/RelatieModel.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        /// </summary>
+        public string InkoopBoekingUri { get; set; }
+
+        /// <summary>
+        /// Verwijzing naar de verkoopboekingen voor de relatie
+        /// </summary>
+        public string VerkoopBoekingUri { get; set; }
+    }
+
+    public class EmailVersturen
```
Comment (henrybeen) [inline SnelStart.B2B.Client/Operations/Relaties/RelatieModel.cs]: This class should ideally be transfered to it's own file

File: SnelStart.B2B.Client/Operations/Response.cs [hunk 2]
```diff
@@ -4,8 +4,8 @@ namespace SnelStart.B2B.Client.Operations
 {
     public class Response
     {
-        internal HttpStatusCode HttpStatusCode { set; get; }
-        internal string ResponseBody { set; get; }
+        public HttpStatusCode HttpStatusCode { set; get; }
```
Comment (henrybeen) [inline SnelStart.B2B.Client/Operations/Response.cs]: I can see why, but this is something that we might not want to expose directly. I'll take note of this

**Comment by henrybeen** [comment]
Fixes #2

File: SnelStart.B2B.Client/Operations/Response.cs (hunk 2 above)
Comment (floris85) [reply SnelStart.B2B.Client/Operations/Response.cs]: how do you suggest i check for a valid response if this is not accessable?

Comment (henrybeensnelstart) [reply SnelStart.B2B.Client/Operations/Response.cs]: I think we have to expose that in a neater way, honoring information hiding. F.e. without exposing http related error codes. But thats a different issue

**Pull request merged**